docs(command): fix custom command node example code issue (fix #188)

AnzhiZhang opened the pull request:
测试截图
![image](https://user-images.githubusercontent.com/37402126/177997778-7de51cbc-dd8c-4457-bf2a-96d374abe8c6.png)

Fallen-Breath commented:
Your implementation does not work with multiple spaces divided arguments, e.g. `!!mypoint 1   2 3`

AnzhiZhang commented:
thx for fixed the issue, and you forgot to change comments in localize docs due to add a new line in the english doc(see my file changes).

AnzhiZhang commented:
> Your implementation does not work with multiple spaces divided arguments, e.g.
> 
> ```
> !!mypoint 1   2 3
> ```

This code could solve this problem and works perfectly:

```python
    try:
        split_text = re.split(' ', text)
        char_read = 0
        args = []
        for i in split_text:
            if i == '':
                char_read += 1
            else:
                args.append(i)
                char_read += len(i) + 1
                if len(args) == 3:
                    break
        coords = list(map(float, args))
        if len(coords) < 3:
            raise IncompletePoint(text)
    except ValueError:
        raise IllegalPoint(text)
    else:
        return ParseResult(coords, char_read)
```

![image](https://user-images.githubusercontent.com/37402126/179512883-071ee26a-11aa-4865-9f5b-4f03f8ae1cda.png)

---

A important thing is this code could save about 40%:

```python
import re
import timeit
from mcdreforged.api.command import *


class IllegalPoint(CommandSyntaxError):
    def __init__(self, char_read: int):
        super().__init__('Invalid Point', char_read)


class IncompletePoint(CommandSyntaxError):
    def __init__(self, char_read: int):
        super().__init__('Incomplete Point', char_read)


def parse1() -> ParseResult:
    try:
        split_text = re.split(' ', text)
        char_read = 0
        args = []
        for i in split_text:
            if i == '':
                char_read += 1
            else:
                args.append(i)
                char_read += len(i) + 1
                if len(args) == 3:
                    break
        coords = list(map(float, args))
        if len(coords) < 3:
            raise IncompletePoint(text)
    except ValueError:
        raise IllegalPoint(text)
    else:
        return ParseResult(coords, char_read)


def parse2() -> ParseResult:
    total_read = 0
    coords = []
    for i in range(3):
        total_read += len(text[total_read:]) - len(
            command_builder_util.remove_divider_prefix(text[total_read:]))
        value, read = command_builder_util.get_float(text[total_read:])
        if read == 0:
            raise IncompletePoint(total_read)
        total_read += read
        if value is None:
            raise IllegalPoint(total_read)
        coords.append(value)
    return ParseResult(coords, total_read)


for text in ['1 2 3', '1  2  3', '1.0  2.2  3.3', '1 2 3 a']:
    time1 = timeit.timeit(parse1, number=100000)
    time2 = timeit.timeit(parse2, number=100000)
    print(time1 / time2)
```

![image](https://user-images.githubusercontent.com/37402126/179513880-18cef87c-b0d6-4882-918e-1ba7c389a3ad.png)

Therefore, I am personally like this implemention. I am happy to open a new PR if you like that!

Fallen-Breath commented:
Time efficient is not considered if the implementation sacrifice readability or maintainability

Fallen-Breath commented:
If you stare at execution efficiency, there are tons of things in MCDR can be optimized a lot, but MCDR is never a cpu intensive program, don't sacrifice more important things for unworthy optimizations
